nio/rooms.py

```diff
@@ -82,6 +84,8 @@ def __init__(self, room_id, own_user_id, encrypted=False):
         self.canonical_alias = None   # type: Optional[str]
         self.topic = None             # type: Optional[str]
         self.name = None              # type: Optional[str]
+        self.parent = set()           # type: Set[str]
```

Comment:
```suggestion
        self.parents = set()           # type: Set[str]
```
I'm thinking plural `parents` makes sense considering that a room is expected to be able to support multiple; but do let me know if I'm thinking about this wrongly